app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -60,6 +60,13 @@ def promote
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_view_versions/:id/republish_repositories", N_("Forces a republish of the version's repositories' metadata.")
+    param :id, :identifier, :desc => N_("Content view version identifier"), :required => true
+    def republish_repositories
+      task = async_task(::Actions::BulkAction, ::Actions::Katello::Repository::MetadataGenerate, @version.repositories, :force => true)
```

Comment:
yeah, this is a common problem in the foreman-tasks page.  I was contemplating changing this to have a new  ContentViewVersion::GenerateMetadata Task rather than using the bulk actions.  let me do that real quick